src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,8 @@
 package mate.academy.lib;
 
-public @interface Inject {
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
 
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Inject {
```

Comment:
same here, you can use Target. I don't know if you have to, though, but this makes it more specified

Reply:
✅